Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -1516,6 +1507,248 @@ def broadcast_message():
         return_error('No messages were sent. Verify that the inputs are correct.')
 
 
+def shutdown_agents_request(query, agent_id, group_id):
+
+    endpoint_url = 'agents/actions/shutdown'
+    filters = {}
+
+    if query:
+        filters['query'] = query
+    if agent_id:
+        filters['ids'] = agent_id
+    if group_id:
+        filters['groupIds'] = group_id
+    if not (agent_id or group_id):
+        return_error('Expecting at least one of the following arguments to filter by: agent_id, group_id.')
+    payload = {
+        'filter': filters
+    }
+
+    response = http_request('POST', endpoint_url, data=json.dumps(payload))
+    if response.get('errors'):
+        return_error(response.get('errors'))
+    else:
+        return response
+
+
+def shutdown_agents():
+    """
+    Sends a shutdown command to all agents matching the input filter
+    """
+    query = demisto.args().get('query', '')
+
+    agent_id = argToList(demisto.args().get('agent_id'))
```

Comment:
check what happens in case both are provided

Reply:
what about this one ?

Reply:
You can provide both.